docs/platform/README.md

```diff
@@ -0,0 +1,49 @@
+# Serverless Platform (Beta)
+
+The Serverless Platform is currently in experimental beta. If you'd like to participate in the beta, simply follow the instructions below.
+
+## Set-Up
+
+Make sure you have Node.js installed and run:
+
+```sh
+$ npm i serverless -g
+```
+
+Then, check the version to make sure you are using V1.20.0, or later:
+
+```sh
+$ serverless -v
+```
+
+## Usage
+
+First, log in to the Serverless platform in via the CLI
+
+```sh
+$ sls login
```

Comment:
Would be nice if we could replace this with `serverless` (rather than `sls`) since `sls` is just a convenience method and might confuse new users.

Furthermore we're already using `serverless -v` above.